### ISSUE TITLE
The highlighter is not case insensitive

### ISSUE THREAD

**Issue by neura1io** [bassjobsen/Bootstrap-3-Typeahead#356]
Even though the matcher is case insensitive, the highlighter is not.

This can be fixed by adding an 'i' to the end of all the regexes in the highlighter function.

**Comment by ftedin** [comment]
Still not fixed? any workaround? Thanks!

**Comment by jaysee** [comment]
Yes, I confirm the issue, you can fix it by adding the `i` flag to the regex used by the default highlighter
in file [`bootstrap3-typeahead.js`](https://github.com/bassjobsen/Bootstrap-3-Typeahead/blob/master/bootstrap3-typeahead.js) on line [`316`](https://github.com/bassjobsen/Bootstrap-3-Typeahead/blob/fc2d5dce4a5406d9b924224e395d5647918707c7/bootstrap3-typeahead.js#L316),  update the code
```var reg = new RegExp(text, 'g');```
by 
```var reg = new RegExp(text, 'gi');```

This should be fixed or be an option to enable this!
